src/jetstream/go.mod

```diff
@@ -117,59 +111,46 @@ replace (
 	k8s.io/sample-apiserver => k8s.io/kubernetes/staging/src/k8s.io/sample-apiserver v0.0.0-20191001043732-d647ddbd755f
 )
 
+require github.com/cloudfoundry-community/go-cfenv v1.18.0
```

Comment:
I don't think we can do this. Part of the fallout from the security incident with the `cloudfoundry-community` org was that we had to remove all references to that org in our code. What errors are we getting from trying to use our fork?

Reply:
this is causing a type collision [here](https://github.com/cloud-gov/stratos/blob/main/src/jetstream/main.go#L111-L113)

Reply:
we can fix this with cloud-gov/cf-common#1

Reply:
updated - we no longer rely on cf community code